code/modules/mob/living/carbon/examine.dm

```diff
@@ -61,8 +61,8 @@
 			disabled += body_part
 		missing -= body_part.body_zone
 		for(var/obj/item/embedded as anything in body_part.embedded_objects)
-			var/stuck_wordage = embedded.is_embed_harmless() ? "stuck to" : "embedded in"
-			. += span_boldwarning("[t_He] [t_has] [icon2html(embedded, user)] \a [embedded] [stuck_wordage] [t_his] [body_part.plaintext_zone]!")
+			var/stuck_wordage = embedded.embed_data.is_harmless() ? "stuck to" : "embedded in"
+			. += span_boldwarning("[t_He] [t_has] [icon2html(embedded, user)] \a <a href='?src=[REF(src)];embedded_object=[REF(embedded)];embedded_limb=[REF(body_part)]' class='danger'>[embedded]</a> [stuck_wordage] [t_his] [body_part.plaintext_zone]!")
```

Comment:
Suggestion (because I did this for my own project) 

Only show the href if examined within x tiles (3 or 4)

Reply:
Also why is there a random `<class='danger'>` in the middle of the text? The entire string is already `boldwarning`, which is red, so it's just saying "make my red, red"

Reply:
Also also, another suggestion

Don't use boldwarning if it's a harmless embed, it's a little silly to see huge "THIS GUY IS DYING" text over "he has tape stuck to him"